app/uk/gov/hmrc/ngrpropertylinkingfrontend/views/SingleSearchResultView.scala.html

```diff
@@ -87,8 +78,17 @@ <h1 class="govuk-heading-l">@messages("singleSearchResultPage.title", postcode)<
             Showing <strong>@pageBottom</strong> to <strong>@pageTop</strong> of <strong>@totalProperties</strong> items.
             <a href= @searchAgainUrl class="govuk-link">@messages("singleSearchResultPage.details.link")</a>
         </p>
-        @sortBySelect
-        @saveAndContinueButton(msg = messages("singleSearchResultPage.button"), isStartButton = false)
+        <div class="govuk-form-group">
+            <label class="govuk-label govuk-!-font-weight-bold" for="sortBy">
+                @messages("singleSearchResultPage.sortBy.label")
+            </label>
+            <select class="govuk-select" id="sortBy" name="sortBy">
+                @for(v <- sortingSelectItems) {
+                    <option value="@v.value" @selected(v)>@v.text</option>
+                }
+            </select>
+            @saveAndContinueButton(msg = messages("singleSearchResultPage.button"), isStartButton = false)
+        </div>
```

Comment:
Could you please explain as to why this has been changed from the twirl component to regular html? This does not match the coding style within the service. Looking at the ticket description, the goal was to remove the options that were not needed. So why did we need to change the entire component as well when it originally worked with the other options